content/about/feature-stages/index.md

```diff
@@ -26,29 +26,18 @@ within the project, not to the project as a whole. Here is a high level descript
 
 Below is our list of existing features and their current phases. This information will be updated after every monthly release.
 
-### Traffic management
+### Networking
```

Comment:
We use Traffic Management everywhere in the doc set, I don't think we should rename this here unless we do a massive sweep through the docs.

Reply:
Done